core/common/cli_deal_stat.hpp

```diff
@@ -0,0 +1,25 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#pragma once
+#include "api/rpc/json.hpp"
+
+namespace fc {
+  using api::StorageMarketDealInfo;
+  using codec::json::Set;
+  using codec::json::Value;
+  using markets::storage::StorageDeal;
+
+  struct CliDealStat {
```

Comment:
Maybe `CliDealState` ?
```suggestion
  struct CliDealState {
```

Reply:
And rename the file if it is true

Reply:
"Stat" is statistic not state

Reply:
Ок